ipfs/ipfs.go

```diff
@@ -8,12 +8,18 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-type IPFS struct {
+type IPFS interface {
+	Add(data []byte) (string, error)
+
+	Get(cid string) ([]byte, error)
+}
+
```

Comment:
Let's add `var _ IPFS = &ipfs{}` to implement the interface.

Reply:
OK I fixed it(https://github.com/medibloc/panacea-oracle/pull/90/commits/608e1f47c232c695a81da5f8fbe3b0ec750947b8)